gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-debug/pom.xml

```diff
@@ -16,8 +16,8 @@
     limitations under the License.
 
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
do you have a particular editorconfig ?

Reply:
no especialy, and even if I do, I didn't touch xml format